cmd/main.go

```diff
@@ -79,3 +86,32 @@ func updateHelp(names []string, c *cobra.Command) {
 		c.Example = strings.ReplaceAll(c.Example, "kubectl "+name, "kapply "+name)
 	}
 }
+
+// cachingRESTClientGetter caches the RESTMapper so every call to
```

Comment:
Eventually, we might want to put this in its own file in another package.

Reply:
Ack